jdaviz/configs/cubeviz/cubeviz.yaml

```diff
@@ -40,5 +40,5 @@ viewer_area:
       - container: row
         viewers:
           - name: Spectrum
-            plot: specviz-profile-viewer
+            plot: cubeviz-profile-viewer
```

Comment:
this allows for the cubeviz instance of specviz to have different tools (including the new slice tool) as well as a new `slice_indicator` property.  Does this have any other unintended consequences that would cause issues?

Reply:
I just checked and we don't explicitly access this configuration key in our notebook, so this shouldn't be an issue for us.  But changes to the yaml config like this are exactly the kind of thing that could impact us, so thanks for checking!